app/models/manageiq/providers/datawarehouse_manager.rb

```diff
@@ -1,5 +1,8 @@
 module ManageIQ::Providers
   class DatawarehouseManager < BaseManager
+    has_many :datawarehouse_nodes, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :advanced_settings, :as => :resource, :dependent => :destroy
```

Comment:
@simon3z do we want to have reports made for this provider with those values (version, and health status data) Currently they are not available in reports.

[1]https://gitter.im/ManageIQ/manageiq/providers?at=586bc1f6db9cafe9183b3981